bootstrap/common.sh

```diff
@@ -31,7 +31,7 @@ verify_prereqs() {
 check_terraform_version() {
   local IFS='.'
   local current_version_string="${2:-$( terraform --version | awk 'NR==1 {print $2}' )}"
-  local requirement_version_string=${1:-0.6.12}
+  local requirement_version_string=${1:-0.6.10} # This is to support Rackspace. Access IPs are not properly exported by the Terraform Openstack provider in versions > 0.6.10. See https://github.com/hashicorp/terraform/issues/5358
```

Comment:
Agreed, just wanted to be able to test it. Thought about adding a Rackspace specific terraform version check function (0.6.8-0.6.10 are acceptable atm), but given that this bug has been identified, fixed and will be released in 0.6.13, I wouldn't worry too much about it. When 0.6.13 is released we can probably use that as a requriement. See https://github.com/hashicorp/terraform/issues/5358